AirCasting/CoreData/PersistenceController+Database.swift

```diff
@@ -123,6 +123,29 @@ extension PersistenceController: SessionUpdateable {
             }
         }
     }
+    
+    func updateNotesPhotosLocations(urls: [(url: URL, noteNumber: Int)], for session: SessionUUID, completion: ((Error?) -> Void)?) {
+        let context = self.editContext
+        context.perform {
+            do {
+                let request = NSFetchRequest<NoteEntity>(entityName: "NoteEntity")
+                let predicate = NSPredicate(format: "session.uuid == %@ AND number IN %@", session.rawValue, urls.map(\.noteNumber))
+                request.predicate = predicate
+                let notes = try context.fetch(request)
+                urls.forEach { noteInfo in
```

Comment:
I not quite a fun of the name `urls` as long as it contains a tuple where noteNumber also exists

Reply:
I can't think of a better name, do you have a suggestion for how to change it?